src/mahoji/lib/abstracted_commands/blackjackCommand.ts

```diff
@@ -0,0 +1,211 @@
+import { channelIsSendable } from '@oldschoolgg/toolkit';
+import { ActionRowBuilder, ButtonBuilder, ButtonStyle, ChatInputCommandInteraction } from 'discord.js';
+import { Time } from 'e';
+import { CommandResponse } from 'mahoji/dist/lib/structures/ICommand';
+import { Bank } from 'oldschooljs';
+import { toKMB } from 'oldschooljs/dist/util';
+
+import { awaitMessageComponentInteraction } from '../../../lib/util';
+import { handleMahojiConfirmation } from '../../../lib/util/handleMahojiConfirmation';
+import { deferInteraction } from '../../../lib/util/interactionReply';
+import { mahojiParseNumber } from '../../mahojiSettings';
+
+const returnButtons = [
+	new ActionRowBuilder<ButtonBuilder>().addComponents([
+		new ButtonBuilder({
+			label: 'Hit',
+			style: ButtonStyle.Secondary,
+			customId: 'hit'
+		}),
+		new ButtonBuilder({
+			label: 'Stand',
+			style: ButtonStyle.Secondary,
+			customId: 'stand'
+		})
+	])
+];
+
+interface Card {
+	suit: string;
+	value: string;
+}
+
+// Define constants for suits and values
+const suits = ['Hearts', 'Diamonds', 'Clubs', 'Spades'];
+const values = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'Jack', 'Queen', 'King', 'Ace'];
```

Comment:
It MIGHT make formatting the game easier if you use a single char for card values (and suits?).

eg. 10 = 'T', J, K , Q, A

It would be REALLY cool to dynamically generate the result like slots. You don't have to draw the entire card, just a simple box around painted unicode letters, 3♠, A♥  etc. Paint spades/clubs black, and hearts/diamonds red. T♣

Look at slots for simple graphics stuff, and the bank code has examples of drawing text (including unicode symbols) onto canvas)